src/output/outputMod.F90

```diff
@@ -1823,17 +1858,93 @@ subroutine storeSurfsolInBuffer(sps, buffer, nn, blockID,   &
        !================================================================
 
     case (cgnsCp)
-
        ! Factor multiplying p-pInf
-
        fact = two/(gammaInf*pInf*MachCoef*MachCoef)
 
        do j=rangeFace(2,1), rangeFace(2,2)
           do i=rangeFace(1,1), rangeFace(1,2)
              nn = nn + 1
-             buffer(nn) = fact*(half*(pp1(i,j) + pp2(i,j)) - pInf)
```

Comment:
![check_cp](https://user-images.githubusercontent.com/36549388/165360598-4c8b6ad2-cec9-4ed3-9826-35575710439b.png)
Yeah, good point. I am sure @Aagaard87 has been very diligent with his derivation, but I double checked to be safe. The Cp curves above are from the MACH-Aero tutorial wing, red is from this PR and green the current `main`. Perfect match at all stations

Reply:
I checked the formulations and both are equivalent for _most_ use cases. However, moving away from using MachCoef breaks some functionality where we can prescribe it separately. This is used for a few cases, sounds like including TS: https://github.com/mdolab/adflow/blob/main/adflow/pyADflow.py#L3015-L3025 

To avoid breaking this functionality, I suggest we modify this part. We can compute the original factor's components outside the loop using machcoef and then in the loop we can keep including the rotational term corrections for the denominator. 

I can probably make this change myself, or give you instructions on how to fix it. Whichever you prefer @marcomangano